llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -438,7 +444,11 @@ static OperandInfo getOperandInfo(const MachineInstr &MI,
   case RISCV::VWMACC_VX:
   case RISCV::VWMACCSU_VV:
   case RISCV::VWMACCSU_VX:
-  case RISCV::VWMACCUS_VX: {
+  case RISCV::VWMACCUS_VX:
+  // Vector Single-Width Fractional Multiply with Rounding and Saturation
+  // Destination EEW=2*SEW and EMUL=2*EMUL. Source EEW=SEW and EMUL=LMUL.
+  case RISCV::VSMUL_VV:
```

Comment:
vsmul doesn't produce a 2*sew result. It does internally and then shift and truncates it to sew

Reply:
Sorry, this wasn't supposed to be a part of this patch anyway. Thanks for the info though.